src/resharper/imports/evaluator-manifest.yaml

```diff
@@ -0,0 +1,71 @@
+queries:
+  issueCountByLevel:
+    query: SELECT COUNT(1) FROM Issues WHERE Level = $Level
+requests:
+  totalFilesCount:
+    query: SELECT COUNT(1) AS count FROM FileDetails
+    type: Scalar
+  totalCodeCount:
+    query: SELECT SUM(Code) FROM FileDetails WHERE Language IN ('C#', 'cshtml')
+    type: Scalar
+  totalLanguagesFilesCount:
+    query: SELECT COUNT(1) as count FROM FileDetails WHERE Language IN ('C#', 'cshtml')
+    ranges:
+      <b><span style='color:red'>(TODO) Number of issues is considered to be high</span></b>:
```

Comment:
What this line is supposed to be used for?

Reply:
```html
<p class=MsoNormal>
        <b>Total files found:</b> {{ totalLanguagesFilesCount.value }} - {{ totalLanguagesFilesCount.range }}
    </p>
```
https://github.com/GodelTech/CodeReview.AdHocReview/pull/3/files#diff-84859f6b181a95032d0b7259c0429f1fa510a499fe2f403fe9e515f7da77823cR210